--- a/arch/mips/ath79/machtypes_orig.h	2016-06-05 17:56:11.740594416 +0200
+++ b/arch/mips/ath79/machtypes.h	2016-06-05 17:54:36.692431622 +0200
@@ -41,6 +41,7 @@
 	ATH79_MACH_BSB,			/* Smart Electronics Black Swift board */
 	ATH79_MACH_CAP4200AG,		/* Senao CAP4200AG */
 	ATH79_MACH_CARAMBOLA2,		/* 8devices Carambola2 */
+	ATH79_MACH_CPE210,		/* TP-LINK CPE210 */
 	ATH79_MACH_CPE510,		/* TP-LINK CPE510 */
 	ATH79_MACH_DB120,		/* Atheros DB120 reference board */
 	ATH79_MACH_PB44,		/* Atheros PB44 reference board */
